neqo-udp/src/lib.rs

```diff
@@ -57,63 +52,89 @@ use std::os::fd::AsFd as SocketRef;
 #[cfg(windows)]
 use std::os::windows::io::AsSocket as SocketRef;
 
-pub fn recv_inner(
+pub fn recv_inner<'a>(
     local_address: &SocketAddr,
```

Comment:
If `SocketAddr` is `Copy`, why take this by reference?

Reply:
@martinthomson see https://github.com/mozilla/neqo/pull/2169.

Reply:
Resolved in #2169.